docs/content/users/install/ddev-installation.md

```diff
@@ -112,9 +112,37 @@ It's easiest to think of DDEV installation or upgrade as the installation of a s
     * Installing or upgrading to the latest Docker Desktop for Windows with WSL2 enabled.
     * Installing DDEV inside your distro.
 
-    We'll walk through these in more detail. You may prefer other techniques of installation or may not need some steps, but this is the full recipe:
+    ### WSL2 Scripted Installation
 
-    1. **Chocolatey:** We recommend using [Chocolatey](https://chocolatey.org/install) for installing required Windows apps like mkcert. In an administrative PowerShell, `Set-ExecutionPolicy Bypass -Scope Process -Force; [System.Net.ServicePointManager]::SecurityProtocol = [System.Net.ServicePointManager]::SecurityProtocol -bor 3072; iex ((New-Object System.Net.WebClient).DownloadString('https://chocolatey.org/install.ps1'))`
+    You can do these things manually, or you can do most of it with the provided PowerShell script. 
+    In all cases:
+    
+    1. Install WSL2 with an Ubuntu distro. On a system without WSL2, just run:
+        ```powershell
+        wsl --install
+        ```
+        If that doesn't work for you, see the [manual installation](https://docs.microsoft.com/en-us/windows/wsl/install-manual) and linked [troubleshooting](https://docs.microsoft.com/en-us/windows/wsl/troubleshooting#installation-issues).
```

Comment:
```suggestion
        If that doesn't work for you, see the [manual installation](https://docs.microsoft.com/en-us/windows/wsl/install-manual) and linked [troubleshooting](https://docs.microsoft.com/en-us/windows/wsl/troubleshooting#installation-issues).
 ```

     2. Use `wsl -l -v` to see your distros and which is the default.
 If you don't already have an Ubuntu distro with it set as default, install it with `wsl --install Ubuntu`.